forums/admin.py

```
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from .models import ForumTopic, ForumReply, ForumNotification
+
+
+@admin.register(ForumTopic)
+class ForumTopicAdmin(admin.ModelAdmin):
+    list_display = ('title', 'author', 'course', 'created_at', 'like_count')
+    search_fields = ('title', 'content', 'author__username', 'course__title')
+    list_filter = ('created_at', 'course')
+    prepopulated_fields = {'slug': ('title',)}
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)